src/backend/marsha/core/templates/core/lti_development.html

```diff
@@ -48,6 +48,7 @@ <h2>Iframe LTI setup</h2>
       <input type="text" name="resource_link_id" value="example.com-df7" />
       <input type="text" name="context_id" value="course-v1:ufr+mathematics+0001" />
       <input type="text" name="roles" value="Instructor" />
+      <input type="text" name="oauth_consumer_key" value="oauth_consumer_key" />
```

Comment:
the `value` should indicate that it must be replaced by the real consumer key, don't you think ? The other value can be sent like this but not this one and it should be reflected IMO